.github/workflows/docker.yml

```diff
@@ -46,7 +46,6 @@ jobs:
           image_id=`cat docker_id.log`
           echo $image_id
           echo "Adjusting permissions so we can access docker logs..."
-          sudo chmod -R +x /var/lib/docker/
```

Comment:
hmmm don't know why logs would need a `+x` but confused why this is there in the first place,

Reply:
hahah, it's moved now :)